typings/primordials.d.ts

```diff
@@ -428,6 +428,9 @@ declare namespace primordials {
   export const StringPrototypeToWellFormed: UncurryThis<typeof String.prototype.toWellFormed>
   export const StringPrototypeValueOf: UncurryThis<typeof String.prototype.valueOf>
   export const StringPrototypeReplaceAll: UncurryThis<typeof String.prototype.replaceAll>
+  // TODO for when Node supports SuppressedError
+  // export import SuppressedError = globalThis.SuppressedError;
+  // export const SuppressedErrorPrototype: typeof SuppressedError.prototype;
```

Comment:
```suggestion
```

Reply:
I committed your suggestion. But I should have first asked why? Do you think such comments do not add value?